en/micro-integrator/docs/references/connectors/file-connector/file-connector-example.md

```diff
@@ -0,0 +1,146 @@
+# File Connector Example
+
+The File Connector allows you to connect to different file systems and perform various operations. The File Connector uses the Apache Commons VFS I/O functionalities to execute operations.
```

Comment:
Shall we link Apache Commons VFS  to the relevant page at Apache?

Reply:
Done